web-platform/functions/src/index.ts

```diff
@@ -2,11 +2,11 @@ import admin from "firebase-admin";
 import functions, { Change, EventContext } from "firebase-functions";
 import { DocumentSnapshot } from "firebase-functions/v1/firestore";
 import { v4 as uuidv4 } from "uuid";
-import { useAuthentication } from "./authentication";
-import { useCors } from "./cors";
-import { studies } from "./studies";
+import { useAuthentication } from "./authentication.js";
+import { useCors } from "./cors.js";
+import { studies } from "./studies.js";
 import { isDeepStrictEqual } from "util";
-import * as gleanPings from "./glean";
+import * as gleanPings from "./glean.js";
 import assert from "assert";
```

Comment:
@aaga ISTR you running into something similar, the tests fail (node complains it can't find the module) unless I add the `.js` suffix, do you know why this is happening? I'm guessing it has to do with one of the newer deps we're importing for the waitlist implementation, if you don't know offhand I'll try to bisect the changes so we can at least know what's causing it.

Reply:
Have you tried adding type: "module" to package.json? We should not be required to add .js to module imports.

Reply:
I believe it already is `type: "module"` in https://github.com/mozilla-rally/rally/blob/eece8547ad5f0fdb43b156247089b1c9d5b026e6/web-platform/functions/package.json#L7

This only seems to be a problem for tests, so it might have to do with how our test runner is hooked up. I don't know why it's a new issue suddenly though, I'll try to bisect the changes and figure it out.

Reply:
It looks like it worked ok in 32de7e0dbe1528c86542fdbd05232ab6f2788c3d and the only substantive change I see there is adding the new modules.